narwhals/_arrow/series.py

```diff
@@ -221,6 +221,10 @@ def n_unique(self) -> int:
         count_unique = pc.count(unique_values, mode="all")
         return count_unique.as_py()  # type: ignore[no-any-return]
 
+    def round(self, decimals: int = 0) -> Self:
+        pc = get_pyarrow_compute()
+        return pc.round(self._native_series, ndigits=decimals)  # type: ignore[no-any-return]
```

Comment:
here we are returning a `pyarrow.ChunkedArray`, instead we need a `ArrowSeries` (as the typing righly says)